bedrock/privacy/templates/privacy/index.html

```diff
@@ -83,7 +83,13 @@ <h1 itemprop="name">{{ doc.h1.string }}</h1>
         <li class="policy-firefox-os"><a href="{{ url('privacy.notices.firefox-os') }}">{{ _('Firefox OS') }}</a></li>
         <li class="policy-firefox-cloud"><a href="{{ url('privacy.notices.firefox-cloud') }}">{{ _('Firefox Cloud Services') }}</a></li>
         <li class="policy-marketplace"><a href="https://marketplace.firefox.com/privacy-policy">{{ _('Firefox Marketplace') }}</a></li>
-        <li class="policy-firefox-focus"><a href="{{ url('privacy.notices.firefox-focus') }}">{{ _('Firefox Focus') }}</a></li>
+        <li class="policy-firefox-focus">
+        {% if LANG == 'de' %}
+          <a href="{{ url('privacy.notices.firefox-klar') }}">Firefox Klar</a>
```

Comment:
I threw this in as a nicety to send `de` users directly to the `-klar` URL. I could make the URL a variable with `set` to make things a little cleaner/not introduce an unwrapped string.